### PR TITLE
add and delete review

### LINKED ISSUE #66: Duplicate Review Error does not show up on shelter page
Need to fix Ajax request in review.js file.

### PR DESCRIPTION
**Pull request by ctedtsen**
Added the forms and routes to add a reviews and delete a review. Also added the first 10 users, 2 of which are admin users.

### CODE REVIEW

File: src/views/shelter/single.handlebars [hunk 1]
```diff
@@ -13,23 +13,47 @@
...
+    <label id="nameLabel">
+        Reviewer Name
+        <input type="text" id="name" name="name" placeholder="Reviewer Name"/>
+    </label>
+    <label id="reviewLabel">
+        Review
+        <input type="text" id="review" name="review" placeholder="Review"/>
+    </label>
+    <label id="ratingLabel">
+        Review
```
Comment (Jasmine211) [inline src/views/shelter/single.handlebars]: Please change from Review to Rating on line 50

**Review by Jasmine211 — CHANGES_REQUESTED** [review]
Looks good! I did find a bug, where a non-registered user would still be able to add reviews to the shelter page, but not to the review collection. However, this should be resolved once I push the user authentication changes.

File: src/views/shelter/single.handlebars (hunk 1 above)
Comment (ctedtsen) [reply src/views/shelter/single.handlebars]: Just fixed it, thanks!

**Review by Jasmine211 — APPROVED** [review]
Thanks, looks good!

**Pull request merged**